### ISSUE TITLE
pptp missing /bin/ip

### ISSUE THREAD

**Issue by jluttine** [NixOS/nixpkgs#31468]
## Issue description

PPTP seems to miss `/bin/ip`. Not sure if it's critical though.

### Steps to reproduce

When I try to connect to PPTP VPN with the network manager, I see the following line in `journalctl -u network-manager`:

```
NetworkManager[1087]: sh: /bin/ip: No such file or directory
```

Also, if I just run `sudo pptp domain.com`, I see

```
$ sudo pptp domain.com
sh: /bin/ip: No such file or directory
sh: /bin/ip: No such file or directory
```

## Technical details

* System: (NixOS: `nixos-version`, Ubuntu/Fedora: `lsb_release -a`, ...)
```
$ nixos-version
18.03.git.66b63d2 (Impala)
```
* Nix version: (run `nix-env --version`)
```
$ nix-env --version
nix-env (Nix) 1.11.15
```
* Nixpkgs version: (run `nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion`)
```
$ nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion
"18.03.git.66b63d2f5a4"
```
* Sandboxing enabled: (run `grep build-use-sandbox /etc/nix/nix.conf`)
```
$ grep build-use-sandbox /etc/nix/nix.conf
build-use-sandbox = true
```

**Comment by ryantm** [comment]
Hi.

Thanks for the report.

It looks like this was fixed by https://github.com/NixOS/nixpkgs/commit/30c739f7a1ba0b176cf6805f97248c804012a046 

Can you update nixpkgs to the latest and try again?

Ryan

**Comment by pSub** [comment]
It was fixed for me with 30c739f7a1ba0b176cf6805f97248c804012a046.
